pycodestyle.py

```diff
@@ -1794,6 +1794,7 @@ def expand_indent(line):
     >>> expand_indent('        \t')
     16
     """
+    line = line.rstrip('\n')
```

Comment:
do we have to look out for `\r\n` (windows newlines)?

Reply:
I think probably not, python normalizes newlines at source read time?  OTOH this doesn't cost much and I doubt it can ever be wrong to strip `\r` so so let's do it too.

Reply:
python2 does not (still supported here sadly) -- and python3 only does in text mode and if newline translation is enabled (it is by default)